mkdocs.yml

```diff
@@ -33,7 +33,8 @@ nav:
   - Home: index.md
   - Using HyP3:
     - using.md
-    - Vertex: using/vertex.md
+    - Vertex:
+      - using/vertex.md
```

Comment:
I think this actually should be fine the way it was; it may just have been mkdocs serve acting up on my laptop again. For some reason, it wouldn't render, but I wonder if it's not actually a problem with the code.